### PR TITLE
pair basebackup start log message with corresponding end one of the same severity

### PR DESCRIPTION
**Pull request by LizardWizzard**
Currently, there is only a message about basebackup start, so by looking at logs, it is not clear whether it is successfully finished or not

### CODE REVIEW

File: pageserver/src/page_service.rs [hunk 1]
```diff
@@ -536,7 +536,7 @@ impl PageServerHandler {
             basebackup.send_tarball()?;
         }
         pgb.write_message(&BeMessage::CopyDone)?;
-        debug!("CopyDone sent!");
+        info!("basebackup CopyDone sent!");
```
Comment (SomeoneToIgnore) [inline pageserver/src/page_service.rs:539]: > basebackup

I would imagine, 

> info_span!("basebackup", timeline = %timelineid, tenant = %tenantid, lsn = field::Empty);

should already append this info?

**Comment by LizardWizzard** [comment]
@SomeoneToIgnore, another log message happen here https://github.com/zenithdb/zenith/blob/main/pageserver/src/basebackup.rs#L86, and before that, we may not know the exact lsn to take basebackup from. Maybe it is better to add two messages, one at the very beginning without the lsn, and then goes this message, and a finish one after that

**Comment by SomeoneToIgnore** [comment]
> one at the very beginning without the lsn, and then goes this message, and a finish one after that

Yeah, or another one into `send_tarball` (if we stick to my initial "move"  proposal), but I like your way more.
Just feels weird a bit to have a `CopyDone sent!` in the logs (even with the span data), since it does not make much sense compared to the `taking basebackup lsn={}, prev_lsn={}` one 🙂

**Comment by LizardWizzard** [comment]
@SomeoneToIgnore see the updated version :)

**Review by SomeoneToIgnore — APPROVED** [review]

**Pull request merged**